deepchem/feat/tests/test_dmpnn_featurizer.py

```diff
@@ -0,0 +1,160 @@
+# flake8: noqa
+"""
+Test for DMPNN Featurizer class
+"""
+
+from deepchem.feat.base_classes import MolecularFeaturizer
+from deepchem.feat.molecule_featurizers.dmpnn_featurizer import DMPNNFeaturizer, GraphConvConstants
+from rdkit import Chem
+import pytest
+import numpy as np
+
```

Comment:
While using Pytest is fine, I think using a TestDMPNNFeaturizer class inheriting from unittest.Testcase might be more in line with how Deepchem featurizer tests have been written till now.

You can maybe look at the tests for MAT or MolGan for example.

```python3
class TestDMPNNFeaturizer(unittest.Testcase):
    def ....():
         ..........
```

Reply:
ok, modify unit tests accordingly.